airsonic-main/src/main/resources/org/airsonic/player/i18n/ResourceBundle_en.properties

```diff
@@ -367,6 +367,8 @@ personalsettings.paginationsize=Number of related albums/directories to show ini
 personalsettings.lastfmenabled=Register what I'm playing at <a href="https://www.last.fm/" target="_blank">Last.fm</a>
 personalsettings.lastfmusername=Last.fm username
 personalsettings.lastfmpassword=Last.fm password
+personalsettings.listenbrainzenabled=Register what I'm playing at <a href="https://listenbrainz.org/" target="_blank">ListenBrainz</a>
```

Comment:
There's no point in putting web links in the help messages, because they won't be clickable anyway (if you move the mouse away from the help icon, the tooltip will disappear).  Also, the help should only contain simple formatting tags, and only when necessary.

Reply:
It's not inside a tooltip, but on the personal settings web page.

Reply:
Ah, gotcha.  In that case, if an external link is needed, please keep in mind #712